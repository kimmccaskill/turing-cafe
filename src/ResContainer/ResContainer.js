import React from 'react';
import Reservation from '../Reservation/Reservation.js'
import './ResContainer.css';

const ResContainer = ({ reservations }) => {
  const appendedReservations = reservations.map(reservation => {
    return <Reservation details={reservation} />
  });
  return (
    <section>
      {appendedReservations}
    </section>
  )
}

export default ResContainer;