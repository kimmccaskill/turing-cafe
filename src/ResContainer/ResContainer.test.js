import React from 'react';
import ResContainer from './ResContainer';
import { shallow } from 'enzyme';

describe('ResContainer', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
    <ResContainer 
        reservations={[{
          "id": 1,
          "name": "Christie",
          "date": "12/29",
          "time": "7:00",
          "number": 12
    }]}
    />)
  })

  it('should be an instance of ResContainer component', () => {
    expect(wrapper).toMatchSnapshot();
  });
})