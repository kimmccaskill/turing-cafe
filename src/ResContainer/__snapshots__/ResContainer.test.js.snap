// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResContainer should be an instance of ResContainer component 1`] = `
<section
  className="res-container"
>
  <Reservation
    details={
      Object {
        "date": "12/29",
        "id": 1,
        "name": "Christie",
        "number": 12,
        "time": "7:00",
      }
    }
  />
</section>
`;
